packages/server/src/api/todo/adx/repo.ts

```diff
@@ -120,6 +120,7 @@ export default function (server: Server) {
     try {
       await repoDiff.processDiff(db, repo, diff)
     } catch (err) {
+      logger.info({ did, err }, 'failed to index batch write')
```

Comment:
Not sure if we want to get to this level of granularity just yet, but this is the kind of thing that may fit nicely under `logger.warn()` or `logger.error()`.  On the flipside of that, there may be some logs that could be downgraded from info the `logger.debug()`.  Just a general thing, probably no need to worry about it in this PR which is already a huge life improvement.

Reply:
good call. i changed these to `warn` & I dropped some of the duplicate ones in `db` fns down to `debug` (I think those were the main culprit)